README.md

```diff
@@ -39,4 +41,28 @@ CBBR is primarily built for planning coordination and information purposes only
 2. `./02_cbbr.sh` to normalize the agency and community district fields
 3. `./03_spatial.sh` to geocode the dataset
 4. `./04_export.sh` to export the dataset as csv
-5. `./05_archive.sh` to archive the cbbr to EDM postgresDB
\ No newline at end of file
+5. `./05_archive.sh` to archive the cbbr to EDM postgresDB
+
```

Comment:
Do we want to be specifc about setting up the pyenv environments here?

Reply:
clarified in [233de9d](https://github.com/NYCPlanning/db-cbbr/pull/31/commits/233de9d0e7c8da324d061a833880535e2bf27e92)